packages/experimental/Shimmer/src/Shimmer.types.ts

```diff
@@ -1,114 +1,156 @@
-import { SvgProps, ImageProps } from 'react-native-svg';
-import { ImageURISource, ColorValue } from 'react-native';
+import { SvgProps } from 'react-native-svg';
+import { ColorValue, ViewProps } from 'react-native';
+import { IBackgroundColorTokens } from '@fluentui-react-native/tokens';
 
 export const shimmerName = 'Shimmer';
+
 /**
- * Specifies the possible type of the shimmer elements: Rect, Circle.
+ * Specifies the possible types of the shimmer elements: Rect, Circle.
  * Required when building more than one element at a time.
  */
 export type ShimmerElementType = 'rect' | 'circle';
+
 /**
- * Represents the shimmer element
+ * Properties necessary to define a circular Shimmer element.
  */
-export interface ShimmerElement {
+export interface ShimmerCircleElement {
   /**
-   * Type of the shimmer element
+   * Shimmer element type.
    */
-  type?: ShimmerElementType;
+  type: 'circle';
+
   /**
-   * Specifies the element width, non-op for circle element
-   * @defaultValue '200'
+   * Radius of the circle element.
+   * @defaultValue 12
    */
-  width?: number;
+  radius?: number;
+
   /**
-   * Specifies the shimmer view height. For circle element type, height is divided by 2 to get the radius for Svg
-   * @defaultValue '100'
+   * Note: cx and cy should be optional properties [or removed], with relative positioning being the default [or only] positioning mechanism.
+   * Note: The behavior of element intersections is undefined.
```

Comment:
If we're going to assert this, I feel we should add open a GitHub issue and put the # in the comment so this note doesn't just dissappear.